src/telemetry/reporter.ts

```diff
@@ -3,30 +3,16 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 import * as vscode from 'vscode';
-import TelemetryReporter from 'vscode-extension-telemetry';
 
-export let reporter: TelemetryReporter;
-
-export class Reporter extends vscode.Disposable {
-    constructor(ctx: vscode.ExtensionContext) {
-        super(() => reporter.dispose());
-
-        const packageInfo = getPackageInfo(ctx);
-        if (packageInfo) {
-            reporter = new TelemetryReporter(packageInfo.name, packageInfo.version, packageInfo.aiKey);
-        }
-    }
-}
-
-interface IPackageInfo {
+export interface IPackageInfo {
     name: string;
     version: string;
     aiKey: string;
 }
 
-function getPackageInfo(context: vscode.ExtensionContext): IPackageInfo | undefined {
+export function getPackageInfo(context: vscode.ExtensionContext): IPackageInfo | undefined {
```

Comment:
nit: I'd rename this file and move it somewhere else. It doesn't fit under 'telemetry/reporter' anymore